lib/order.rb

```diff
@@ -0,0 +1,25 @@
+class Order
+  attr_reader :dishes
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+    @dishes = {}
+  end
+
+  def add(dish, quantity)
+    raise "Apologies, #{dish} is not available" unless @menu.dish_available?(dish)
+  
+    @dishes[dish] = quantity
+  end
+
+  def bill
```

Comment:
Could we make this a private method to achieve more encapsulation? Are there other methods that could be private?